src/controllers/auth.controller.ts

```diff
@@ -0,0 +1,122 @@
+import User from "../database/models/user.model";
+import Session from "../database/models/session.model";
+import bcrypt from 'bcrypt';
+import { createClient } from "redis";
+import jwt from 'jsonwebtoken';
+
+const client = createClient();
```

Comment:
connection should be managed from a single file only and then client should be exported